synfig-core/src/CMakeLists.txt

```diff
@@ -183,6 +183,8 @@ else()
 
 		## TODO: move to module where it is actually required
 		pkg_check_modules(PANGO REQUIRED pango)
```

Comment:
I don't know why Pango is here. `synfig-core` does not depend on Pango.

By the way:
```
$ git grep PANGO_INCLUDE_D
synfig-core/src/CMakeLists.txt: find_path(PANGO_INCLUDE_DIR pango/pango.h)
synfig-core/src/modules/lyr_freetype/CMakeLists.txt:target_include_directories(lyr_freetype PRIVATE ${PANGO_INCLUDE_DIRS})
synfig-core/src/synfig/CMakeLists.txt:        ${PANGO_INCLUDE_DIRS}
```
and

```
$ git grep PANGO_LIBRAR
synfig-core/src/CMakeLists.txt: find_library(PANGO_LIBRARY pango-${PANGO_LIB_SUFFIX})
synfig-core/src/CMakeLists.txt: set(PANGO_LIBRARIES ${PANGO_LIBRARY} ${PANGO_CAIRO_LIBRARY} ${PANGO_WIN32_LIBRARY})
synfig-core/src/modules/lyr_freetype/CMakeLists.txt:target_link_libraries(lyr_freetype synfig ${CAIRO_LIBRARIES} ${PANGO_LIBRARIES} ${PANGOCAIRO_LIBRARIES} ${FT_LIBRARIES} ${FONT_CONFIG_LIBRARIES})

```

Reply:
There are still pango/cairo dependencies in synfig-core (for vcpkg), they will be moved separately.

Reply:
Really? Wow I can't see why :)